imp/TokenSet.cs

```diff
@@ -154,7 +154,7 @@ public class TokenSet : ITokenSet {
 
 }; /* end m2c_token_name_table */
 
-        public static const int segmentCount = (Enum.GetNames(typeof(Token)).Length / 32) + 1;
+        public const int segmentCount = (Enum.GetNames(typeof(Token)).Length / 32) + 1;
 

```

Comment:
Constants and variables within an implementation of a class should always be private. If a client of the class needs to read the constant or variable, then there should be a public function method to return its value. If a client of the class needs to write to a variable, then there should be a public mutator method to overwrite its value. Such public methods will then need to be added to the public interface of the class.